teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1923 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #53.        test_pairs['prediction'] = pd.Series(predictions)
</pre><p>I might well be wrong here but could you please check if you really need the <code>pd.Series()</code> part (i.e. can you pass the list directly; <code>predictions</code> is a list, right?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>Indeed</p>